src/sonic-config-engine/minigraph.py

```diff
@@ -1178,6 +1178,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
 
     if bool(results['PEER_SWITCH']):
         results['DEVICE_METADATA']['localhost']['subtype'] = 'DualToR'
+        if len(results['PEER_SWITCH'].keys()) > 1:
```

Comment:
What is the underlying cause of the two peers entities? Is this coming from the GeminiPeeringLink property? If so, can we use the hostname from the graph to pick the correct peer?

Reply:
We should never have two peer entities, it would only occur if there were an error in the minigraph (if it had two of the peering link tags).